src/implementations/wrap/wrap.ts

```diff
@@ -193,19 +219,40 @@ export const wrap = async ({
 
   // Phase 1: For each document, read content, digest and write to file
   const schema = await loadSchema(schemaPath);
-  const individualDocumentHashes = await digestDocument(inputPath, intermediateDir, version, unwrap, schema);
+  const individualDocumentHashes = await digestDocument(
+    inputPath,
+    // if we dont batch document we can directly write to the destination folder.
+    batched ? intermediateDir : outputPath,
```

Comment:
Can explain this in code?

const individualDocumentsDir = batched ? intermediateDir : outputPath;

Reply:
The current comment doesn't help ? I didn't explain the second part though. If not batched need to write to a temp folder for later to compute the melee root